Backpack/src/main/java/net/skyscanner/backpack/dialog/internal/BpkDialogImpl.kt

```diff
@@ -19,103 +19,114 @@
 package net.skyscanner.backpack.dialog.internal
 
 import android.app.Dialog
+import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.graphics.drawable.InsetDrawable
+import android.os.Build
+import android.util.DisplayMetrics
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.view.ViewGroup.LayoutParams
 import android.view.Window
+import android.view.WindowManager
 import android.widget.ImageView
 import android.widget.TextView
 import androidx.annotation.LayoutRes
 import net.skyscanner.backpack.R
 import net.skyscanner.backpack.button.BpkButton
 import net.skyscanner.backpack.dialog.BpkDialog
+import kotlin.math.min
+
+class BpkDialogImpl(
```

Comment:
is this mainly merging the two dialog classes? any breaking changes here from an api perspective? it doesn't look like given that it's internal, but wanted to double check

Reply:
I don't think so - @xo0ps will be able to confirm as he made this change.

Reply:
@marianeum yes this PR merges two implementations of BpkDialog because they were using different xml layouts. By simplifiying the layouts I was able to remove specific implementations and merge them. Also nothing from api perspective is affected.